applications/StructuralMechanicsApplication/custom_strategies/custom_builder_and_solver/explicit_builder_and_solver.hpp

```diff
@@ -0,0 +1,461 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics 
+//
```

Comment:
Do you really need to create your own Builder And Solver for the explicit?, right @RiccardoRossi ?

Reply:
on this i second @loumalouomega 

you could simplify the strategy to avoid scheme, space and builder and solver